js/apps/admin-ui/src/clients/add/NewClientForm.tsx

```diff
@@ -66,33 +91,6 @@ export default function NewClientForm() {
     }
   };
 
-  const forward = async (onNext?: () => void) => {
-    if (!(await trigger())) {
-      return;
-    }
-    if (!isFinalStep()) {
-      setStep(step + 1);
-    }
-    onNext?.();
-  };
-
-  const isFinalStep = () =>
-    protocol === "openid-connect" ? step === 2 : step === 1;
```

Comment:
don't we still need this logic so that we have one more step in case of "openid-connect"?

Reply:
@edewit nope, the step is hidden within the individual `WizardStep` component now, using `isHidden`:

```
            <WizardStep
              name={t("capabilityConfig")}
              id="capabilityConfig"
              key="capabilityConfig"
              isHidden={protocol === "saml"}
            >
              <CapabilityConfig protocol={protocol} />
            </WizardStep>
```